.github/workflows/build.yml

```diff
@@ -242,6 +249,24 @@ jobs:
         name: ${{ matrix.os }} build
         path: ppsspp/
 
+    - name: Create release
+      if: startsWith(github.ref, 'refs/tags/') && runner.os == 'macOS' && matrix.extra == 'test'
+      run: |
+        cd ppsspp || exit 1
+        mv PPSSPPSDL.zip PPSSPPSDL-${GITHUB_REF##*/}.zip || exit 1
```

Comment:
Should this have `macOS` in the name? E.g. `PPSSPPSDL-macOS-v1.2.3.zip` What is the convention for the release filenames?

Reply:
Actually did this.